chia/consensus/block_creation.py

```diff
@@ -213,6 +219,7 @@ def create_foliage(
         additions_root = addition_merkle_set.get_root()
         removals_root = removal_merkle_set.get_root()
 
+        # I thought I did this already
```

Comment:
Maybe need to pull from main again? This should be a hash and not treehash right?